cmd/update/volume_move.go

```diff
@@ -133,17 +159,21 @@ func (c *volumeMoveCommand) runWithCtx(ctx context.Context, cmd *cobra.Command,
 		// Failing to do so is not a big problem as api-manager will eventually
 		// sync the labels and bring the number down to match the pvc but it may
 		// take a while to act (up to 1h).
+		//
+		// function defers the variable values from when created and not when it
+		// gets called thus timeout, original replica count, etc need to be
+		// up-to-date here or obtained from the API when needed.
 		defer func() {
-			updatedVol, err := c.client.GetVolume(ctx, nsID, vol.ID)
+			updatedVol, err := c.getVolume(ctx, nsID)
 			if err != nil {
-				c.display.TextMessage(ctx, c.writer, "failed to get updated volume config")
+				c.display.TextMessage(ctx, c.writer, fmt.Sprintf("Failed to restore original replica count: failed to get updated volume config: %s", err.Error()))
 				return
 			}
 
 			currentReplicaCount := len(updatedVol.Replicas)
 
 			if currentReplicaCount-originalReplicaCount != 1 {
-				c.display.TextMessage(ctx, c.writer, "replica count of volume has been changed outside of move operation")
+				c.display.TextMessage(ctx, c.writer, "Replica count has been changed outside of move operation, aborting automatic deletion. Volume replicas needs to be managed manually.")
```

Comment:
What does this involve? Can we tell users what to do as part of this message or should we have some knowledge base article about it?

Reply:
The user will read something like this (and in this order):

"Creating replica on [destination] node"
"Syncing replica..."
"Replica count has been changed outside of move operation, aborting automatic deletion. Volume replicas needs to be managed manually." - This message

So I hope either the user:
- understands the last message means the first step was not reverted
- reads the volume replicas need to be managed somehow and then checking the replicas understand what happened and adjusts the replica count as he needs

Reply:
Sorry, I should've been more specific. I meant what does `Volume replicas needs to be managed manually.` involve. Telling the user to select the correct replica and adjust the replica count is probably too much for a log message 😓